imageanalysis/CMakeLists.txt

```diff
@@ -385,22 +385,19 @@ install (FILES
 
 
 add_custom_target(
-    modify_variant_h
-    COMMAND cp ${CMAKE_SOURCE_DIR}/${CASA_CODE_PATH}/stdcasa/variant.h ${CMAKE_SOURCE_DIR}/imageanalysis
-    COMMAND sed -e 's/throw[(]error[)]/noexcept(false)/g' ${CMAKE_SOURCE_DIR}/imageanalysis/variant.h > ${CMAKE_SOURCE_DIR}/imageanalysis/variant.h.tmp
-    COMMAND mv ${CMAKE_SOURCE_DIR}/imageanalysis/variant.h.tmp ${CMAKE_SOURCE_DIR}/imageanalysis/variant.h
+    patch_stdcasa_variant
+    COMMAND sed -i 's/throw[(]error[)]//g' ${CMAKE_SOURCE_DIR}/${CASA_CODE_PATH}/stdcasa/variant.h ${CMAKE_SOURCE_DIR}/${CASA_CODE_PATH}/stdcasa/StdCasa/variant.cc
```

Comment:
This sed command doesn't work on MacOS: `command a expects \ followed by text`.

It can be fixed for MacOS by adding `''` after -i, but then that doesn't work for Linux. 

Perhaps it could be done in this basic way:
> if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
> COMMAND sed -i '' 's/throw[(]error[)]//g' ${CMAKE_SOURCE_DIR}/${CASA_CODE_PATH}/stdcasa/variant.h ${CMAKE_SOURCE_DIR}/${CASA_CODE_PATH}/stdcasa/StdCasa/variant.cc
> endif
> if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
>     COMMAND sed -i 's/throw[(]error[)]//g' ${CMAKE_SOURCE_DIR}/${CASA_CODE_PATH}/stdcasa/variant.h ${CMAKE_SOURCE_DIR}/${CASA_CODE_PATH}/stdcasa/StdCasa/variant.cc
> endif

Or maybe you know a more clever single line universal way? 
Otherwise the old three line way (cp / sed -e / mv) works too.